src/ServerlessOffline.js

```diff
@@ -20,10 +21,17 @@ export default class ServerlessOffline {
   #lambda = null
   #serverless = null
 
-  constructor(serverless, cliOptions) {
+  constructor(serverless, cliOptions, v3Utils) {
     this.#cliOptions = cliOptions
     this.#serverless = serverless
 
+    if (v3Utils) {
+      this.log = v3Utils.log
```

Comment:
Should we use this notation of a specific version? When we release v4 it will become a bit misleading

Reply:
> When we release v4 it will become a bit misleading

I think not necessary. by semver v3 utils may be available in v4, v5 and so on, it's just clear they won't be available in v2.

More to the point name could be `v3PlusUtils`, but I'm not sure if it's not even more confusing. Do you have better name in mind?

Reply:
I was thinking about just calling it `utils` but I guess extra information is useful as it brings context. I don't really have a better suggestion here so we can keep it the way it is :+1: I'll assume that we'll use this notation across different plugins as well